turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -268,7 +268,11 @@ impl StandardBroadcastRun {
 
         // 2) Convert entries to shreds and coding shreds
         let is_last_in_slot = last_tick_height == bank.max_tick_height();
-        let reference_tick = bank.tick_height() % bank.ticks_per_slot();
```

Comment:
If we are changing this I would rather to change it to:
```rust
let reference_tick = last_tick_height
    .saturating_add(bank.ticks_per_slot())
    .saturating_sub(bank.max_tick_height());
```

Reply:
Hmm, I don't think that will work exactly as is. The issue is somewhat analogous to tick-height being 1-indexed whereas reference-tick needs to be 0-indexed. That being said, I still like the idea and think I can make a change to avoid branching in favor of arithmetic. See https://github.com/anza-xyz/agave/pull/4644/commits/5dec84996c478ad8c933b5510cb923061a668b82